mRemoteV1/App/NativeMethods.cs

```diff
@@ -381,6 +384,11 @@ public static int HIWORD(IntPtr value)
         /// <summary>
         /// 
         /// </summary>
+        public const int WM_CHAR = 0x102;
```

Comment:
Include a summary from MS about this message type:

"The WM_CHAR message contains the character code of the key that was pressed. "